app/src/main/AndroidManifest.xml

```diff
@@ -2,8 +2,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="com.stevesoltys.seedvault"
-    android:versionCode="7"
-    android:versionName="1.0.0">
+    android:versionCode="29000001"
+    android:versionName="10-1.0">
```

Comment:
2 lines changed, from 30 and 11 respectively.

Reply:
@stevesoltys last release was 1.0.0, guessing this one could be 10-1.0 ?

We should bump this given the changes we've done though.

Reply:
If we are putting out a release for the backported changes, `10-1.1.0` sounds good to me. I think we should keep version bumps for releases in separate PRs though.

Reply:
So to answer your question.. maybe `10-1.0.0` for this PR.